src/org/usfirst/frc/team467/robot/Robot.java

```diff
@@ -81,6 +83,12 @@ public void disabledInit() {
 
 	public void disabledPeriodic() {
 		LOGGER.trace("Disabled Periodic");
+		String[] autoList = {"None", "Just_Go_Forward", "Left_Switch_Only", "Left_Basic", "Left_Advanced", "Left_Our_Side_Only",
+				"Center", "Right_Switch_Only", "Right_Basic", "Right_Advanced", "Right_Our_Side_Only"};
+		NetworkTableInstance tableInstance = NetworkTableInstance.getDefault();
+		NetworkTable table  = tableInstance.getTable("SmartDashboard");
+		table.getEntry("Auto List").setStringArray(autoList);
+		LOGGER.info("Selected Auto Mode: " + SmartDashboard.getString("Auto Selector", "None"));
```

Comment:
We should move this into a method in one of the autonomous classes, something like initAutoModeMenu().